com.ibm.wala.cast/source/java/com/ibm/wala/cast/loader/CAstAbstractLoader.java

```diff
@@ -62,31 +59,24 @@ public CAstAbstractLoader(IClassHierarchy cha) {
     this(cha, null);
   }
 
+  private Set<Warning> messagesFor(ModuleEntry module) {
+    return errors.computeIfAbsent(module, (key) -> new HashSet<>());
+  }
+
   public void addMessage(ModuleEntry module, Set<Warning> message) {
```

Comment:
Rename to `addMessages`?  (Since we're modifying this code anyway)

Reply:
OK. I will take care of this in a group of updates for this pull request.